crates/re_viewer_context/src/component_ui_registry.rs

```diff
@@ -188,85 +200,34 @@ pub struct ComponentUiRegistry {
     /// Ui method to use if there was no specific one registered for a component.
     fallback_ui: ComponentUiCallback,
 
+    /// Pure viewers
     component_uis: BTreeMap<ComponentName, ComponentUiCallback>,
```

Comment:
```suggestion
    /// Pure viewers. TODO(#6779): to be fully replaced by component_singleline_editors/component_multiline_editors
    component_uis: BTreeMap<ComponentName, ComponentUiCallback>,
```

that said, do we have consensus on this? (I think so, just checking:))

Reply:
I'm not sure that is the best TODO to point to
![image](https://github.com/rerun-io/rerun/assets/1148717/d63cc639-ece5-498c-bc05-0fe5c8e2039f)